### DISCUSSION TITLE
GLSP V2.1.0 and VSCode Integration does not work

### DISCUSSION THREAD

**Discussion by rsoika** [eclipse-glsp/glsp#1208] [General]
After I have migrated my project to version 2.1.0 I try now to migrate the VS-Code-Integration part.

I followed the latest [glsp-vscode-integration/example/workflow](https://github.com/eclipse-glsp/glsp-vscode-integration/tree/master/example/workflow) and try to do all in the same way.

The `app.ts` file in the example looks like this:

```javascript
class WorkflowGLSPStarter extends GLSPStarter {
    createContainer(...containerConfiguration: ContainerConfiguration): Container {
        return createWorkflowDiagramContainer(...containerConfiguration);
    }
}

export function launch(): void {
    new WorkflowGLSPStarter();
}
```

When I try to compile it using yarn I got the following compiler error:

```
src/app.ts(32,43): error TS2556: A spread argument must either have a tuple type or be passed to a rest parameter.

```

What does this error message mean? Can I it be an issue that I am mainly still working with [this pull request](https://github.com/eclipse-glsp/glsp-client/pull/306). Which is not yet official?

**Comment by tortmayr** [comment]
@rsoika 
I''m closing this discussion for now as it was based on a nightly version/ PR. If the probelm persists with 2.1.0 please open a new discussion.
